samples/powerpoint/basics/insert-image.yaml

```diff
@@ -0,0 +1,62 @@
+id: ''
+gist: ''
+source: POWERPOINT
+author: Microsoft
+name: insert-image
+description: Inserts an image to the current slide.
+script:
+  content: |
+    $('#insert').click(run);
+
+    async function run() {
+        var base64EncodedImageStr = getImageAsBase64String();
+
+        await Office.context.document.setSelectedDataAsync(base64EncodedImageStr, {
```

Comment:
I removed the **await** and **async** keywords. I only put them in because I noticed that the generated YAML file set the language to typescript and there wasn't actually anything TypeScripty in the code. It is now pure JS again, which I guess is technically also TS, but JMHO it seems odd to call it typescript in the yaml.